tests/series/arithmetic_test.py

```diff
@@ -15,18 +15,25 @@
         ("__add__", 1, [2, 3, 4]),
         ("__sub__", 1, [0, 1, 2]),
         ("__mul__", 2, [2, 4, 6]),
-        ("__truediv__", 2, [0.5, 1.0, 1.5]),
+        ("__truediv__", 2.0, [0.5, 1.0, 1.5]),
```

Comment:
Related to the comment above, since the data are all integers, if other is also an integer, then the data type is maintained. This is different from how polars and pandas handle it